modules/navigation/3d/nav_mesh_generator_3d.cpp

```diff
@@ -939,6 +939,7 @@ void NavMeshGenerator3D::generator_bake_from_source_geometry_data(Ref<Navigation
 			nav_polygons.push_back(nav_indices);
 		}
 	}
+	p_navigation_mesh->set_polygons(nav_polygons);
```

Comment:
This line is not needed as the newer set_data() is used below to set the vertices and polygons arrays in a thread-safe way so they stay in sync.